projects/igniteui-angular/src/lib/services/exporter-common/base-export-service.ts

```diff
@@ -26,6 +27,11 @@ export enum ExportRecordType {
     HeaderRecord = 5,
 }
 
+export enum ColumnType {
```

Comment:
Enum's name does not match the values. If we keep 'ColumnType' name, the values should be DataColumn and GroupColumn I guess.